python/ray/job_config.py

```diff
@@ -1,12 +1,68 @@
 import uuid
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
 
+import ray.cloudpickle as pickle
+from ray._private.structured_logging.utils import LOG_MODE_DICT
 from ray.util.annotations import PublicAPI
 
 if TYPE_CHECKING:
     from ray.runtime_env import RuntimeEnv
 
 
+@PublicAPI(stability="alpha")
+class LoggingConfig:
+    def __init__(self, log_config: Union[dict, str] = "TEXT", log_level: str = "INFO"):
```

Comment:
log_config -> dict_config? to make it clear that we use python dictConfig to configure the logging.